LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
[OPTIONAL]:

The license was a great addition to your project, I suggest you to keep it! :)